packages/app/src/components/HelloWorld.vue

```diff
@@ -1,45 +1,17 @@
 <template>
   <div>
-    <h1>{{ msg }}</h1>
-    <button @click="count++">count is: {{ count }}</button>
-    <p>
-      Edit <code>components/HelloWorld.vue</code> to test hot module
-      replacement.
-    </p>
-    <pre>
-      {{ users }}
-    </pre>
+    <NicksPetList />
   </div>
 </template>
 
 <script lang="ts">
-import { defineComponent, onMounted, ref } from 'vue';
-
-import feathers from '@/lib/feathers';
-import { User } from '@project-leek/commons';
+import { defineComponent } from 'vue';
+import NicksPetList from './NicksPetList.vue';
 
 export default defineComponent({
   name: 'HelloWorld',
-
-  props: {
-    msg: {
-      type: String,
-      default: 'Hexe',
-    },
-  },
-
-  setup() {
-    const count = ref(0);
-    const users = ref<User[]>([]);
-
-    onMounted(async () => {
-      // users.value = await feathers.service('users').find();
-    });
-
-    return {
-      count,
-      users,
-    };
+  components: {
```

Comment:
You use your HelloWorld component only to create your pet component.
Maybe create your pet component directly in the App.Vue.